.github/workflows/release.yml

```diff
@@ -91,7 +91,7 @@ jobs:
                 name: browsermon_win_x64
                 path: artifact/
     
-    build_linux_x86:
+    build_linux_x64:
```

Comment:
until now we were building 32-bit binaries ?

Reply:
It was a typo, The job was making x64 binaries always.